### PR TITLE
Document connection protocols and ciphers docs

### PR DESCRIPTION
**Pull request by erzetpe**

### CODE REVIEW

File: docs/home/SECURITY.md [hunk 1]
```diff
@@ -150,74 +154,85 @@ The list does not include ports that are bound to the loopback interface (localh
 ### Connection protocols and ciphers used by components in Epiphany
 
 Below you can find list of cipersuites and protocols used for communication set in Epiphany on per component basis. This
-doesn't contains information about deprecated components: Apache Ignite, Istio, Hashicorp Vault.
+doesn't contain information about deprecated components: Apache Ignite, Istio, Hashicorp Vault.
```
Comment (atsikham) [inline docs/home/SECURITY.md]: image

File: docs/home/SECURITY.md [hunk 2]
```diff
@@ -150,74 +154,85 @@ The list does not include ports that are bound to the loopback interface (localh
...
     - protocols:  
       TLSv1.1,TLSv1.2
 
     - ciphersuites:  
-      ECDHE-RSA-AES128-GCM-SHA256 ECDHE-RSA-AES256-GCM-SHA384 ECDHE-RSA-AES128-SHA256 ECDHE-RSA-AES256-SHA384 ECDHE-RSA-AES256-SHA ECDHE-RSA-AES128-SHA AES256-GCM-SHA384 AES128-GCM-SHA256 AES256-SHA256 AES128-SHA256 AES256-SHA AES128-SHA
+      ECDHE-RSA-AES128-GCM-SHA256 ECDHE-RSA-AES256-GCM-SHA384 ECDHE-RSA-AES128-SHA256 ECDHE-RSA-AES256-SHA384
+      ECDHE-RSA-AES256-SHA ECDHE-RSA-AES128-SHA AES256-GCM-SHA384 AES128-GCM-SHA256 AES256-SHA256 AES128-SHA256
+      AES256-SHA AES128-SHA
 
 9. Grafana:
```
Comment (atsikham) [inline docs/home/SECURITY.md]: There is something strange with the list order

**Review by atsikham — CHANGES_REQUESTED** [review]

File: docs/home/SECURITY.md (hunk 1 above)
Comment (atsikham) [reply docs/home/SECURITY.md]: This is about 'deprecated' components

File: docs/home/SECURITY.md [hunk 3]
```diff
@@ -16,7 +16,8 @@ antivirus/antimalware solution.
 
 ### Users and roles created by epiphany
 
-By default Epiphany is creating user operations that is used to connect to machines with admin rights on every machine. This setting can be changed in Epiphany yaml configuration files.
+By default, Epiphany creates user operations that is used to connect to machines with admin rights on every machine.
```
Comment (rafzei) [inline docs/home/SECURITY.md]:
```suggestion
By default, Epiphany creates user 'operations' that is used to connect to machines with admin rights on every machine.
```

File: docs/home/SECURITY.md [hunk 4]
```diff
@@ -16,7 +16,8 @@ antivirus/antimalware solution.
...
+This setting can be changed in Epiphany yaml configuration files.
```
Comment (rafzei) [inline docs/home/SECURITY.md:20]:
```suggestion
This setting can be changed in Epiphany initial yaml manifest.
```
Did you mean that?

File: docs/home/SECURITY.md [hunk 5]
```diff
@@ -147,23 +151,128 @@ The list does not include ports that are bound to the loopback interface (localh
...
+
+    - protocols:  
+      TLSv1.2,TLSv1.1,TLSv1
+
+    - ciphersuites:  
+      Depends on version of Java and for Java versions older than 8u161 on JCE policy file. From update 8u161 JCE policy
+      file is not needed to enable restricted ciphers as all ciphers are enabled by default. Documentation about ciphers
+      is available under [link](https://docs.oracle.com/javase/8/docs/technotes/guides/security/SunProviders.html).
+
+4. Open Distro:
```
Comment (rafzei) [inline docs/home/SECURITY.md:189]: Hmm, could you verify if it supports tlsv1.3? Not sure what OpenSSL version is required/privded.
https://github.com/opendistro-for-elasticsearch/security/blob/86d6292d7b2649363437e558ce3fa71c1a3b2afc/src/main/java/com/amazon/opendistroforelasticsearch/security/ssl/DefaultOpenDistroSecurityKeyStore.java#L751

**Review by rafzei — CHANGES_REQUESTED** [review]

File: docs/home/SECURITY.md (hunk 2 above)
Comment (erzetpe) [reply docs/home/SECURITY.md]: This is the output from the cipherscan tool, I will leave as it is.

File: docs/home/SECURITY.md (hunk 1 above)
Comment (erzetpe) [reply docs/home/SECURITY.md]: Changed to potentially deprecated.

File: docs/home/SECURITY.md (hunk 4 above)
Comment (erzetpe) [reply docs/home/SECURITY.md:20]: This should be done at configuration level.

File: docs/changelogs/CHANGELOG-1.3.md [hunk 6]
```diff
@@ -15,6 +15,10 @@
 - [#2644](https://github.com/epiphany-platform/epiphany/issues/2644) - Add validation to check hostnames for on-prem deployment
 - [#2703](https://github.com/epiphany-platform/epiphany/issues/2703) - Add tests for docker and kubelet cgroup driver
 - [#1076](https://github.com/epiphany-platform/epiphany/issues/1076) - Add sorting entries in the inventory file
+<<<<<<< HEAD
+- [#2814](https://github.com/epiphany-platform/epiphany/issues/2814) - Add description how to enable TLS in Kibana
+=======
+>>>>>>> 48077bb9f3993d41bfb80df8ffabe9cb898a1ca5
```
Comment (atsikham) [inline docs/changelogs/CHANGELOG-1.3.md]: Change

File: docs/home/HOWTO.md [hunk 7]
```diff
@@ -70,6 +70,7 @@
   - [How to enable/disable Epiphany service user](./howto/SECURITY.md#how-to-enabledisable-epiphany-service-user)
   - [How to use TLS/SSL certificate with HA Proxy](./howto/SECURITY.md#how-to-use-tlsssl-certificate-with-ha-proxy)
   - [How to use TLS/SSL certificate with RabbitMQ](./howto/SECURITY.md#how-to-use-tlsssl-certificate-with-rabbitmq)
+  - [How to use TLS/SSL certificate with Kibana](./howto/SECURITY.md#how-to-use-tlsssl-certificate-with-kibana)
```
Comment (atsikham) [inline docs/home/HOWTO.md:73]: Why is it here? Seems to be related to another PR.

**Review by atsikham — CHANGES_REQUESTED** [review]

File: docs/changelogs/CHANGELOG-1.3.md (hunk 6 above)
Comment (erzetpe) [reply docs/changelogs/CHANGELOG-1.3.md]: Fixed.

File: docs/home/SECURITY.md (hunk 5 above)
Comment (erzetpe) [reply docs/home/SECURITY.md:189]: In our configuration it doesn't support, as you need to change template or do such things manually. And here we are mentioning only protocols setup by Epiphany.

File: docs/home/HOWTO.md (hunk 7 above)
Comment (erzetpe) [reply docs/home/HOWTO.md:73]: There is something with history after rebase. Too long to rework right now I think.

File: docs/changelogs/CHANGELOG-1.3.md [hunk 8]
```diff
@@ -15,6 +15,8 @@
 - [#2644](https://github.com/epiphany-platform/epiphany/issues/2644) - Add validation to check hostnames for on-prem deployment
 - [#2703](https://github.com/epiphany-platform/epiphany/issues/2703) - Add tests for docker and kubelet cgroup driver
 - [#1076](https://github.com/epiphany-platform/epiphany/issues/1076) - Add sorting entries in the inventory file
+- [#2814](https://github.com/epiphany-platform/epiphany/issues/2814) - Add description how to enable TLS in Kibana
```
Comment (rafzei) [inline docs/changelogs/CHANGELOG-1.3.md:18]: I think this will be doubled after merge, but If you want to fix it after merge, its up to you

File: docs/home/HOWTO.md (hunk 7 above)
Comment (rafzei) [inline docs/home/HOWTO.md:73]: same

**Review by rafzei — APPROVED** [review]

**Review by atsikham — APPROVED** [review]
